packages/wiz/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.6.1"
+  changes:
+    - description: Update vulnerability data stream
```

Can we have a more informative description? What is the update and how would a user know whether making the upgrade is important for them? Suggest something like...
```suggestion
    - description: Add support for CDR to vulnerability data stream.
```
or
```suggestion
    - description: Improve support for CDR in vulnerability data stream.
```